src/Kronos.WFD.Client/Requests/Timezones/TimezoneRequestBuilder.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Collections.Generic;
+
+namespace Kronos.WFD.Client.Requests
+{
+    internal enum TimezoneRequestQueryParameter
+    {
+        TimezoneName,
+        LocationID,
+        EmployeeID,
+        PersonNumber
+    }
+
+    internal class TimezoneRequestBuilder : BaseRequestBuilder, ITimezoneRequestBuilder
+    {
+        /*
```

Comment:
Rather than having these comments here you can document each corresponding method and the parameters with descriptions here